dipy/align/tests/test_expectmax.py

```diff
@@ -225,66 +243,83 @@ def test_compute_em_demons_step_3d():
         raise AssertionError("Failed for sigma_i_sq == 0 and gradient == 0")
 
-    #Test sigma_i_sq != 0 and gradient == 0
+    # Test sigma_i_sq != 0 and gradient == 0
     try:
-        assert_array_almost_equal(actual[random_labels==3], expected[random_labels==3])
+        assert_array_almost_equal(actual[random_labels == 3],
+                                  expected[random_labels == 3])
     except AssertionError:
         raise AssertionError("Failed for sigma_i_sq != 0 and gradient == 0 ")
 
-    #Test sigma_i_sq != 0 and gradient != 0
+    # Test sigma_i_sq != 0 and gradient != 0
     try:
-        assert_array_almost_equal(actual[random_labels==4], expected[random_labels==4])
+        assert_array_almost_equal(actual[random_labels == 4],
+                                  expected[random_labels == 4])
     except AssertionError:
         raise AssertionError("Failed for sigma_i_sq != 0 and gradient != 0")
 
+
 def test_quantize_positive_2d():
     np.random.seed(1246592)
 
-    num_levels = 11 # an arbitrary number of quantization levels
-    img_shape = (15, 20) # arbitrary test image shape (must contain at least 3 elements)
+    # an arbitrary number of quantization levels
+    num_levels = 11
+    # arbitrary test image shape (must contain at least 3 elements)
+    img_shape = (15, 20)
     min_positive = 0.1
     max_positive = 1.0
     epsilon = 1e-8
 
     delta = (max_positive - min_positive + epsilon) / (num_levels - 1)
-    true_levels = np.zeros((num_levels,), dtype = np.float32)
+    true_levels = np.zeros((num_levels,), dtype=np.float32)
     # put the intensities at the centers of the bins
-    true_levels[1:] = np.linspace(min_positive+delta*0.5, max_positive-delta*0.5, num_levels-1)
-    true_quantization = np.empty(img_shape, dtype = np.int32) # generate a target quantization image
-    random_labels = np.random.randint(0, num_levels, np.size(true_quantization))
+    true_levels[1:] = np.linspace(min_positive + delta * 0.5,
+                                  max_positive - delta * 0.5, num_levels - 1)
+    # generate a target quantization image
+    true_quantization = np.empty(img_shape, dtype=np.int32)
+    random_labels = np.random.randint(0, num_levels,
+                                      np.size(true_quantization))
 
     # make sure there is at least one element equal to 0, 1 and num_levels-1
     random_labels[0] = 0
     random_labels[1] = 1
-    random_labels[2] = num_levels-1
+    random_labels[2] = num_levels - 1
     true_quantization[...] = random_labels.reshape(img_shape)
 
-    noise_amplitude = np.min([delta / 4.0, min_positive / 4.0]) # make sure additive noise doesn't change the quantization result
-    noise = np.random.ranf(np.size(true_quantization)).reshape(img_shape) * noise_amplitude
+    # make sure additive noise doesn't change the quantization result
+    noise_amplitude = np.min([delta / 4.0, min_positive / 4.0])
+    noise = np.random.ranf(np.size(true_quantization)).reshape(
```

Comment:
To shorten the line, we can define a new variable for the array length:
```
    sz = np.size(true_quantization)
    noise = np.random.ranf(sz).reshape(img_shape) * noise_amplitude
```
this way the product appears in the same line. A bit more readable, I think.